server/models/pattern/core/pattern.go

```diff
@@ -43,7 +43,7 @@ type Service struct {
 	Name        string            `yaml:"name,omitempty" json:"name,omitempty"`
 	Type        string            `yaml:"type,omitempty" json:"type,omitempty"`
 	APIVersion  string            `yaml:"apiVersion,omitempty" json:"apiVersion,omitempty"`
-	Namespace   string            `yaml:"namespace" json:"namespace"`
+	Namespace   string            `yaml:"namespace,omitempty" json:"namespace,omitempty"`
```

Comment:
> Was omitempty added for MeshModel components, too?

@leecalcote namespace is a pattern-file contruct and independent of any interface. So it will work for both without discrimination.

Reply:
Oh, very good. :thu